enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -13,13 +13,150 @@
 from enterprise_access.apps.api_client.discovery_client import DiscoveryApiClient
 from enterprise_access.apps.api_client.lms_client import LmsApiClient
 from enterprise_access.apps.content_assignments.tasks import BrazeCampaignSender, _get_assignment_or_raise
-from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+from enterprise_access.apps.subsidy_request.constants import (
+    LearnerCreditAdditionalActionStates,
+    LearnerCreditRequestActionErrorReasons,
+    SubsidyRequestStates
+)
+from enterprise_access.apps.subsidy_request.models import LearnerCreditRequestActions
+from enterprise_access.apps.subsidy_request.utils import get_action_choice, get_error_reason_choice, get_user_message_choice
 from enterprise_access.tasks import LoggedTaskWithRetry
-from enterprise_access.utils import get_subsidy_model
+from enterprise_access.utils import format_traceback, get_subsidy_model
 
 logger = logging.getLogger(__name__)
 
 
+def _get_learner_credit_request_or_raise(learner_credit_request_uuid):
```

Comment:
[Curious] Do we really need this utility? we can directly querry this learner_credit_request_model.objects.get(uuid=learner_credit_request_uuid) at anywhere.